src/tasks/command/command_parser.h

```diff
@@ -0,0 +1,45 @@
+/**
+ * @author  Thomas Haile
+ * @date    2025-05-24
+ *
+ * Command parsing and data structure definitions
+ */
+
+#pragma once
+
+#include "packet.h"
+#include "slate.h"
+#include <stdbool.h>
+#include <stdint.h>
+#include <string.h>
+
+// Command IDs (must start from 1, as 0 indicates "not uploading")

```

Comment:
Hmm, what does "not uploading" mean? Could we not shift NO_OP up to the first and preserve TAKE_PHOTO as id value 1?

Generally, for style, either we assign values to all these enums, or not. But we should be consistent (and I'd prefer explicitly assigning values).